lib/host-manager.js

```diff
@@ -50,11 +50,16 @@ exports.add = function add(config, name, type, path, args) {
     return;
   }
 
-  if(path && !Path.isAbsolute(path)) {
+  if (path && !Path.isAbsolute(path)) {
     path = Path.join(process.cwd(), path);
   }
 
-  config.hosts[name] = { type, path, args };
+  if (tags) {
+    if (typeof tags === 'string') {
+      tags = [tags];
```

Comment:
Isn't this already taken care of with `hostTags = argv.tags.split(',');` (and then `hostTags` is passed in for the `tags` param to this function?